Gruntfile.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(grunt) {
   grunt.initConfig({
     _test_runner: '_mocha',
     _istanbul: 'istanbul cover --dir',
-    _unit_args: '--recursive -t 10000 ./test/',
+    _unit_args: '--recursive -t 10000 ./test/test_request-id.js',
```

Comment:
grunt-fh-build actually has a command to run just 1 file, but it needs a local config anyway: https://github.com/fheng/fh-mbaas/blob/master/Gruntfile.js#L14